### DISCUSSION TITLE
V0.8 RIA UF2 file trashed on GITHUB?

### DISCUSSION THREAD

**Discussion by GlennFR** [picocomputer/community#116] [Q&A]
Hi all.
So, I've finally had time to build my picocomputer - but I can't get the V0.8 RIA firmware to work.  When I copy the file to the pico, the pico disconnects but the LED doesn't light.
If I power-up the system with the 0.8 RIA on the board it immediately crashes with 
`/home/rumble/rp6502/src/littlefs/lfs.c:1346:error: Cor`
The V0.7 firmware copies correctly, the LED lights-up *AND* the picocomputer works fine.
I have downloaded the files from [https://github.com/picocomputer/rp6502/releases] several times on different PCs, always the same pb.
Could somebody test the V0.8 file by downloading it on a system, please?

**Comment by WojciechGw** [comment]
0.8 worked fine for me at first glance
but
RTC time set app doesn't work
any app using graphics mode neither ...

Have I missed something ?
Do I have to recompile all applications for 0,8 ?

I would also add that in the first impulse
I cloned anew the entire rp6502 repository
and compiled my own versions of VGA and RIA.
However, this did not have the expected effect

![1724080181402](https://github.com/user-attachments/assets/319cf66e-9cdd-448d-a1fb-1f7e1dd64cad)

**Comment by netzerohero** [comment]
RIA-ver-0.8 worked for me.  

- I used RIA Version-0.8 image rp6502_ria.uf2 dated 24-July-2024 from the release-zip bundle. 
- (I don't build RIA or VGA firmware images locally.)

1. Note however: I didn't install VGA-ver-0.8.  My understanding (after reviewing rp6502/src repo) is that VGA did not change in move to VGA-ver-0.8 from VGA-ver-0.7.
2. A quick check of graphics - worked for me.
3. I didn't test RTC.
4. I didn't test issue originally reported [here](https://github.com/orgs/picocomputer/discussions/105#discussioncomment-9953783).
5. I use the Debian-Edition of Linux Mint (LMDE 6); not based on either Ubuntu 24.04 or 22.04.

![20240819_170223](https://github.com/user-attachments/assets/916ac2e2-8333-4371-94b1-2fa24579c3a4)

Hope this helps!

**Comment by okwatts** [comment]
I too had this problem when I first tried this on my build. 0.7 worked fine but 0.8 didn't. I now have it working but I can't be sure how I managed it!
I believe I re-downloaded the 0.8 zip file and tried again after installing 0.7 (it was the ria that gave problems). I sympathize with you because I was going to post about my issues but then it worked. I regret I can't remember exactly what I did that caused it to work. Pretty sure it was all on Linux Mint and not a Windows machine.

**Reply by okwatts** [reply]
One other issue I noted was that the USB stick I first used had some kind of security mechanism and it hung the Picocomputer because it was so slow in reading files. I changed to a new USB stick and it works fine.

**Comment by GlennFR** [comment]
So, thank you for your comments and feedback.  I too have retried loading RIA 0.8 on the pico that was alrady flashed with 0.7 and it now works.
The question remains : why does the 0.8 image NOT WORK on a stock, brand-new & unused pi pico ?

**Comment by rumbledethumps** [comment]
I was able to reproduce this. Simply bumping littlefs to the latest version seems to fix it. Released fixed binary as v0.9.
https://github.com/picocomputer/rp6502/releases
